flutter/lib/src/flutter_enricher.dart

```diff
@@ -0,0 +1,185 @@
+import 'dart:async';
+import 'dart:ui';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:sentry/sentry.dart';
+
+typedef WidgetBindingGetter = WidgetsBinding Function();
+
+/// Enriches [SentryEvent]s with various kinds of information.
+/// FlutterEnricher only needs to add information which aren't exposed by
+/// the Dart runtime.
+class FlutterEnricher implements Enricher {
+  FlutterEnricher(
+    this._checker,
+    this._dartEnricher,
+    this._getWidgetsBinding,
+  );
+
+  factory FlutterEnricher.simple(PlatformChecker checker) {
+    return FlutterEnricher(
+      checker,
+      Enricher(),
+      () => WidgetsFlutterBinding.ensureInitialized(),
+    );
+  }
+
+  // We can't use `WidgetsBinding` as a direct parameter
+  // because it must be called inside the `runZoneGuarded`-Integration.
+  // Thus we call it on demand after all the initialization happend.
+  final WidgetBindingGetter _getWidgetsBinding;
+  WidgetsBinding get _widgetsBinding => _getWidgetsBinding();
+  final PlatformChecker _checker;
+  final Enricher _dartEnricher;
+  SingletonFlutterWindow get _window => _widgetsBinding.window;
+  final List<String> _packages = [];
+
+  @override
+  FutureOr<SentryEvent> apply(
+      SentryEvent event, bool hasNativeIntegration) async {
+    // Flutter for Web does not need a special case.
+    // It's already covered by _dartEnricher.
+
+    // First use _dartEnricher.
+    // In case we have even better information available in Flutter
+    // we override what's already given by _dartEnricher.
+    event = await _dartEnricher.apply(event, hasNativeIntegration);
+
+    // If there's a native integration available, it probably has better
+    // information available than Flutter.
+    final device =
+        hasNativeIntegration ? null : _getDevice(event.contexts.device);
+
+    final contexts = event.contexts.copyWith(
+      device: device,
+      runtimes: _getRuntimes(event.contexts.runtimes),
+    );
+
+    // Flutter has a lot of Accessibility Settings available and exposes them
+    contexts['accessibility'] = _getAccessibilityContext();
+
+    // Conflicts with Flutter runtime if it's just called `Flutter`
+    contexts['flutter_context'] = _getFlutterContext();
+
+    contexts['culture'] = _getCulture();
+
+    contexts['packages'] = await _getPackages();
+
+    return event.copyWith(
+      contexts: contexts,
+    );
+  }
+
+  Future<Map<String, dynamic>> _getPackages() async {
```

Comment:
And because we're getting these from the LicenseRegistry, these are only the packages which have a license.

Reply:
I think version is mandatory, https://develop.sentry.dev/sdk/event-payloads/sdk/
so if it does not work, set the version as "unknown"
worth adding a comment to the code about packages not being available because there's no license

Reply:
Isn't the SDK just for the version information of the Sentry SDK?

Reply:
yes, indeed, for the App's dependencies, it'd be `event.modules`